src/Framework/Constraint/Count.php

```diff
@@ -22,11 +21,16 @@ class Count extends Constraint
      */
     private $expectedCount;
 
+    /**
+     * @var null|\SplObjectStorage
+     */
+    private $traversableCounts;
+
     public function __construct(int $expected)
     {
         parent::__construct();
 
-        $this->expectedCount = $expected;
+        $this->expectedCount     = $expected;
```

Comment:
leftover from intermediate commits, can be reverted back

Reply:
Done